site/src/_includes/codepen.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="codepen-link">
+  <form action="http://codepen.io/pen/define" method="POST" target="_blank">
+    <input type="text" name="data" hidden="true"
+      value='{
+        "html":"{{ escape html}}",
+        "js": "{{{ escape js }}}",
+        "css": "{{#if css}}{{{ escape css }}}{{/if}}",
+        "js_external": "https://cdnjs.cloudflare.com/ajax/libs/d3fc/5.2.0/d3fc.bundle.js",
```

Comment:
These hardcoded values are going to trip us up, worth changing to [this pattern](https://github.com/ScottLogic/d3fc/pull/835/files#diff-9b2d0e08433fde5a173fd219bf9ffad3L55)?